demo/browseApi.js

```diff
@@ -1,68 +1,61 @@
 const Ebay = require('../src/index');
 const { clientId, clientSecret } = require('./credentials/index');
 
-let access_token = '';
 let ebay = new Ebay({
     clientID: clientId,
     clientSecret: clientSecret,
-    body: {
-        grant_type: 'client_credentials',
-        scope: 'https://api.ebay.com/oauth/api_scope'
-
-    }
 });
 
 // Getting access token and calling getItem method.
-ebay.getAccessToken()
+ebay.getApplicationToken()
     .then((data) => {
```

Comment:
Shall we keep name as `getAccessToken` since we are using this convention from the start ?.